modules/linear_boltzmann_solvers/b_discrete_ordinates_solver/lbs_discrete_ordinates_solver.cc

```diff
@@ -892,24 +890,26 @@ DiscreteOrdinatesSolver::InitializeSweepDataStructures()
 {
   log.Log() << program_timer.GetTimeString() << " Initializing sweep datastructures.\n";
 
-  // Perform checks
-  {
-    auto& mesh_handler = GetCurrentHandler();
-    auto& mesher = mesh_handler.GetVolumeMesher();
-
-    for (const auto& groupset : groupsets_)
-    {
-      bool no_cycles_parmetis_partitioning =
-        ((mesher.options.partition_type == VolumeMesher::PartitionType::PARMETIS) and
-         (not groupset.allow_cycles_));
-
-      bool is_1D_geometry = options_.geometry_type == GeometryType::ONED_SLAB;
-
-      if (no_cycles_parmetis_partitioning and not is_1D_geometry and (opensn::mpi_comm.size() > 1))
-        throw std::logic_error("When using PARMETIS type partitioning then groupset iterative "
-                               "method must be NPT_CLASSICRICHARDSON_CYCLES or NPT_GMRES_CYCLES");
-    } // for groupset
-  }
+  //  // Perform checks
+  //  {
+  //    auto& mesh_handler = GetCurrentHandler();
+  //    auto& mesher = mesh_handler.GetVolumeMesher();
+  //
+  //    for (const auto& groupset : groupsets_)
+  //    {
+  //      bool no_cycles_parmetis_partitioning =
+  //        ((mesher.options.partition_type == VolumeMesher::PartitionType::PARMETIS) and
+  //         (not groupset.allow_cycles_));
+  //
+  //      bool is_1D_geometry = options_.geometry_type == GeometryType::ONED_SLAB;
+  //
+  //      if (no_cycles_parmetis_partitioning and not is_1D_geometry and (opensn::mpi_comm.size() >
+  //      1))
+  //        throw std::logic_error("When using PARMETIS type partitioning then groupset iterative "
+  //                               "method must be NPT_CLASSICRICHARDSON_CYCLES or
+  //                               NPT_GMRES_CYCLES");
+  //    } // for groupset
+  //  }
```

Comment:
I wanted to ask about this. Is this still relevant? I commented this out and it does not break anything, but wanted to double check with you...

Reply:
@zhardy-lanl @wdhawkins , remind me what the NPT_ prefix is all about. Is this all/deprecated? and also, reply to David's query.

Reply:
Ah, that is VERY old! Stands for **N**eutral **P**article **T**ransport. The prefixes can be removed (or the options eliminated) if unused.

Reply:
So, just to confirm, this is safe to delete. Plus, I can add a patch to remove the options, etc.